src/core/main.js

```diff
@@ -575,6 +575,8 @@ p5.prototype._initializeInstanceVariables = function() {
   };
 
   this._pixelsDirty = true;
+
+  this.downKeys = {}; //Holds the key codes of currently pressed keys
```

Comment:
Might we prefer `this._downKeys` to imply this is a private property?

Reply:
Oh yes , that is better. I will make the changes .
Thanks !

Reply:
looks good!